README.md

```diff
@@ -150,6 +150,95 @@ multipath -ll
 
 3.2. For Fibre Channel, configure the relevant zoning from the storage to the host.
 
+<br/>
+<br/>
+<br/>
+
+# SecurityContextConstraints RequirementsThe operator uses the restricted and privileged SCC for deployments. ### Custom SecurityContextConstraints definition:<details>
```

Comment:
this doesn't look right. please compare it to the source

Reply:
@oriyarde , pls check